### ISSUE TITLE
Media control voice activation - DAC_Usability_Voice_Activation_Issue3

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#188]
On page 60 of the report DAC describe an issue with ID `DAC_Usability_Voice_Activation_Issue3` relating to voice activation of a native `<audio>` element using Dragon software. 

This will need further exploration and testing to identify how we can improve this. I wonder whether the presence of `aria-label` is impacting the experience. 

### Why was this not identified in the in-house testing
It is not a WCAG failure.
